test/unit/csi-daemonset.bats

```diff
@@ -177,17 +177,40 @@ load _helpers
       --set "csi.enabled=true" \
       . | tee /dev/stderr |
       yq -r '.spec.template.spec.containers[0].args[1]' | tee /dev/stderr)
-  [ "${actual}" = "--debug=false" ]
+  [ "${actual}" = "--log-level=info" ]
 
   local actual=$(helm template \
```

Comment:
local is already declared above, we can probably drop the extra declaration here.